src/main/scala/viper/gobra/translator/encodings/StringEncoding.scala

```diff
@@ -292,7 +292,7 @@ class StringEncoding extends LeafTypeEncoding {
         results = Vector(res),
         pres = Vector(pre),
         posts = Vector(),
-        terminationMeasures = Vector(),
+        terminationMeasures = Vector(in.WildcardMeasure(None)(info)),
```

Comment:
the comment at the top of this `val` should be updated correspondingly.
BTW, I'm wondering whether we distinguish between `decreases` and `decreases _` for abstract functions/methods. I guess the only difference occurs when you eventually add a body as the latter case makes it easy to forget about the obligation to prove termination

Reply:
For abstract methods, I think that there is no practical difference. It is true that using `decreases _` makes it easier to forget adding the correct termination measure. At the same time, I prefer using this notation because, unlike the first, it does not assume that it has a constant termination measure - it just assumes that there is some termination measure for this function.